static/audiofeed.yaml

```
@@ -1,3 +1,17 @@
+- title: 'Fantastische Fakten 1'
+  date: 2025-06-30T19:00
+  image: comic-cast.jpg
+  description: |2-
+                  Liebe Zuhörer:innen,
+                  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)